src/main/kotlin/com/terraformation/backend/accelerator/variables/AcceleratorProjectVariableValuesService.kt

```diff
@@ -118,34 +136,75 @@ class AcceleratorProjectVariableValuesService(
               }
             }
             ?.toSet() ?: emptySet()
+    val landUseHectares =
+        StableIds.landUseHectaresByLandUseModel
+            .mapNotNull { (landUseType, stableId) ->
+              getNumberValue(valuesByStableId, stableId)?.let { landUseType to it }
+            }
+            .filter { (landUseType, _) -> landUseType in landUseModelTypes }
+            .toMap()
     val maxCarbonAccumulation = getNumberValue(valuesByStableId, StableIds.maxCarbonAccumulation)
+    val methodologyNumber =
+        getSingleSelectValue(variablesById, valuesByStableId, StableIds.methodologyNumber)
     val minCarbonAccumulation = getNumberValue(valuesByStableId, StableIds.minCarbonAccumulation)
+    val minProjectArea = getNumberValue(valuesByStableId, StableIds.minProjectArea)
     val numNativeSpecies = getNumberValue(valuesByStableId, StableIds.numSpecies)?.toInt()
     val perHectareBudget = getNumberValue(valuesByStableId, StableIds.perHectareEstimatedBudget)
+    val projectArea = getNumberValue(valuesByStableId, StableIds.projectArea)
+    val riskTrackerLink = getLinkValue(valuesByStableId, StableIds.riskTrackerLink)
+    val sdgList =
+        getMultiSelectValue(variablesById, valuesByStableId, StableIds.sdgList)
+            ?.mapNotNull {
+              try {
+                SustainableDevelopmentGoal.forJsonValue(it)
```

Comment:
This block is long enough that using a named lambda parameter instead of `it` would make things clearer.

Reply:
Will do. This was a copy of the existing country variable above, so I'll update that one too